content/docs/other-guides/virtual-dev/getting-started-minikube.md

```diff
@@ -93,7 +93,7 @@ chmod +x setup-minikube.sh
 **KUBEFLOW_TAG** is a tag corresponding to the version to checkout such as {{% kf-latest-version %}} or `master`.
 
 The script asks for some config input as shown below:
-![ConfigInputs](../ConfigInputs.png)
+![ConfigInputs](/content/docs/started/ConfigInputs.png)
```

Comment:
Hallo @mosn thanks for taking this on!

We usually put the images in the `images` directory, tho that's not essential:
https://github.com/kubeflow/website/tree/master/content/docs/images

The path to the image should start at `/docs/` (not at `/content/`).

While you're here, please would you fix the styling of the image too? (There's a bit about images in the [README](https://github.com/kubeflow/website/blob/master/README.md) though that could certainly be expanded.)

Here's an example of an image with styling and path:

```
<img src="/docs/images/consent-screen.png" 
  alt="OAuth consent screen"
  class="mt-3 mb-3 p-3 border border-info rounded">
```

The above example comes from this page:
https://github.com/kubeflow/website/tree/master/content/docs/images